gamemodes/terrortown/gamemode/server/sv_addonchecker.lua

```diff
@@ -274,11 +264,6 @@ addonChecker.curatedList = {
 		reason = "Does not use the TTT2 sidebar system.",
 		type = ADDON_OUTDATED
 	},
-	["652046425"] = { -- Juggernaut Suit by Zaratusa
-		alternative = "1640512667",
```

Comment:
This addon is indeed not for TTT2, the alternative is this one: https://steamcommunity.com/sharedfiles/filedetails/?id=2157829981